index.html

```diff
@@ -51,17 +51,17 @@ <h2>
         Like crunchy cornflakes<br>
         Gold leaves rustle underfoot<br>
         Beauty in decay.<br>
-        <em>By<em> <b>Paul Holmes
+        <em>By<b>Paul Holmes</b></em>
```

Comment:
You have an unclosed `<b>` tag, and the whole author line was supposed to be inside the `<em>` tag